### PR TITLE
Ze permissions(z)

### PR DESCRIPTION
**Pull request by tlemburg**

### CODE REVIEW

File: src/UNL/UCBCN/Manager/Controller.php [hunk 1]
```diff
@@ -205,6 +206,26 @@ public function getCalendars() {
         return $user->getCalendars();
     }
 
+    public function hasPermission($permission_description, $calendar_id) 
+    {
+        switch ($permission_description) {
```
Comment (mfairchild365) [inline src/UNL/UCBCN/Manager/Controller.php]: I guess I'm not sure why you need this.  Can you not just use the permission constants?

Comment (tlemburg) [inline src/UNL/UCBCN/Manager/Controller.php]: Yeah, not my favorite part. I can't use the Permission constants in the template. This was the best idea I had to work around it for now

**Comment by tlemburg** [comment]
Please check for typos, there was a lot of copy-pasted stuff here. I was careful but one may have slipped through

File: src/UNL/UCBCN/Manager/Controller.php (hunk 1 above)
Comment (mfairchild365) [inline src/UNL/UCBCN/Manager/Controller.php]: You totally can!

Comment (tlemburg) [inline src/UNL/UCBCN/Manager/Controller.php]: I'm going to have to disagree with you...

[Tue Jun 23 21:30:32 2015] [error] [client 10.0.2.2] PHP Fatal error:  Class 'Permission' not found in /var/www/html/www/templates/default/html/manager/Calendar.tpl.php on line 31, referer: http://iimlemburg.unl.edu:8008/manager/immediate77/users/s-tlembur1/edit/

Comment (mfairchild365) [inline src/UNL/UCBCN/Manager/Controller.php]: that is what you get when you use the fully qualified namespace?
IE:
```
\UNL\UCBCN\Permission::EVENT_CREATE
```

Comment (kabel) [inline src/UNL/UCBCN/Manager/Controller.php]: Namespaces complicate it a little bit. Best to use the fully-qualified name `\UNL\UCBCN\Permission::`

Comment (tlemburg) [inline src/UNL/UCBCN/Manager/Controller.php]: Oh, ok. Well I can flip that over then. Namespaces are really a silly concept you know

File: src/UNL/UCBCN/Manager/DeleteRecurrence.php [hunk 2]
```diff
@@ -23,6 +24,11 @@ public function __construct($options = array())
             throw new \Exception("That calendar could not be found.", 404);
         }
 
+        $user = Auth::getCurrentUser();
+        if (!$user->hasPermission(Permission::EVENT_EDIT_ID, $this->calendar->id)) {
+            throw new \Exception("You do not have permission to edit events on this calendar.", 403);
```
Comment (mfairchild365) [inline src/UNL/UCBCN/Manager/DeleteRecurrence.php]: should this be delete or edit?

Comment (tlemburg) [inline src/UNL/UCBCN/Manager/DeleteRecurrence.php]: That should be edit because that is removing a single recurrence from a single datetime of an event.

Comment (mfairchild365) [inline src/UNL/UCBCN/Manager/DeleteRecurrence.php]: Okie :)

**Comment by mfairchild365** [comment]
+1 

its looking nice @tlemburg !!!!

**Pull request merged**